app/Http/Controllers/Legacy/Web/ActionsController.php

```diff
@@ -47,7 +47,7 @@ public function store(Request $request)
             'name' => 'required|string',
             'campaign_id' => 'required|integer|exists:campaigns,id',
             'post_type' => 'required|string|in:photo,voter-reg,text,share-social,phone-call',
-            'callpower_campaign_id' => 'required_if:post_type,phone-call|exists:actions,callpower_campaign_id',
+            'callpower_campaign_id' => 'nullable|required_if:post_type,phone-call|integer|unique:actions,id',
```

Comment:
I think we want this to be to be `unique:actions,callpower_campaign_id` right (since that's the column on that table that we want to be unique, not `id`)? If so, we can actually shorten this to `unique:actions` as a shorthand!

Reply:
Let's also make sure to update the rule in the `update` method down below, which is still using `exists`! Since we want to "ignore" the existing action when checking uniqueness, we'll need to follow the instructions under the ["Forcing A Unique Rule To Ignore A Given ID"](https://laravel.com/docs/5.8/validation#rule-unique) heading there.